Index: Assets/Prefabs/ExplosionVFX.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: b87929575dffb6c49b64ff8ec3faa5d7\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/ExplosionVFX.prefab.meta b/Assets/Prefabs/ExplosionVFX.prefab.meta
--- a/Assets/Prefabs/ExplosionVFX.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/ExplosionVFX.prefab.meta	(date 1754430385618)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: b87929575dffb6c49b64ff8ec3faa5d7
+guid: 4c2c7f80f7e56ec4ba781069419d1eea
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/Main Camera.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: dfd1ae625419872458577af010d4bb9d\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/Main Camera.prefab.meta b/Assets/Prefabs/Main Camera.prefab.meta
--- a/Assets/Prefabs/Main Camera.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/Main Camera.prefab.meta	(date 1754430385625)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: dfd1ae625419872458577af010d4bb9d
+guid: dfdc39f3fff346a4a972fa07cd6e7d8f
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/StarField.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 86a961cb599661b47963d0668fbd1935\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/StarField.prefab.meta b/Assets/Prefabs/StarField.prefab.meta
--- a/Assets/Prefabs/StarField.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/StarField.prefab.meta	(date 1754430385642)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 86a961cb599661b47963d0668fbd1935
+guid: 8755ed1d8043d7f418ff7a1853a5b691
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/HighRes.asset.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 3d4c13846a3e9bd4c8ccfbd0657ed847\nNativeFormatImporter:\n  externalObjects: {}\n  mainObjectFileID: 11400000\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/HighRes.asset.meta b/Assets/HighRes.asset.meta
--- a/Assets/HighRes.asset.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/HighRes.asset.meta	(date 1754430385598)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 3d4c13846a3e9bd4c8ccfbd0657ed847
+guid: fe6a32ad3b12234419b4fdcd4aa73293
 NativeFormatImporter:
   externalObjects: {}
   mainObjectFileID: 11400000
Index: Assets/Prefabs/UIOverlay.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 87f4b42386fd2af44b25b722572d1fcd\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/UIOverlay.prefab.meta b/Assets/Prefabs/UIOverlay.prefab.meta
--- a/Assets/Prefabs/UIOverlay.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/UIOverlay.prefab.meta	(date 1754430385646)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 87f4b42386fd2af44b25b722572d1fcd
+guid: 08115a0d58fa6ba4a914d0374e5de4e7
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Particles/Embers (1).prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 7f4473226fdfe5241a37d4e359857821\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Particles/Embers (1).prefab.meta b/Assets/Particles/Embers (1).prefab.meta
--- a/Assets/Particles/Embers (1).prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Particles/Embers (1).prefab.meta	(date 1754430385609)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 7f4473226fdfe5241a37d4e359857821
+guid: c50edc186adb74c419b5cf1860794e13
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Particles/SmokeTrail.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 7a8f4f11b43bc6343b3d77bf7abc11ba\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Particles/SmokeTrail.prefab.meta b/Assets/Particles/SmokeTrail.prefab.meta
--- a/Assets/Particles/SmokeTrail.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Particles/SmokeTrail.prefab.meta	(date 1754430385611)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 7a8f4f11b43bc6343b3d77bf7abc11ba
+guid: 8d542bf5204f45e41a071275feb46a4a
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/SmallExplosion.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: f2996a623a527754b92cee330859cd26\nNativeFormatImporter:\n  externalObjects: {}\n  mainObjectFileID: 100100000\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/SmallExplosion.prefab.meta b/Assets/Prefabs/SmallExplosion.prefab.meta
--- a/Assets/Prefabs/SmallExplosion.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/SmallExplosion.prefab.meta	(date 1754430385631)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: f2996a623a527754b92cee330859cd26
+guid: 45e22938b5cd6c742a985435cca92438
 NativeFormatImporter:
   externalObjects: {}
   mainObjectFileID: 100100000
Index: Assets/Prefabs/SpaceBackground.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: e44175a016a306e4f9d76e593cfc4dab\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/SpaceBackground.prefab.meta b/Assets/Prefabs/SpaceBackground.prefab.meta
--- a/Assets/Prefabs/SpaceBackground.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/SpaceBackground.prefab.meta	(date 1754430385636)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: e44175a016a306e4f9d76e593cfc4dab
+guid: f30900c3cf80d6042a28f7d13e43363d
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/MissileProjectile.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 5615e4673e17c104384d4c8493c5d42b\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/MissileProjectile.prefab.meta b/Assets/Prefabs/MissileProjectile.prefab.meta
--- a/Assets/Prefabs/MissileProjectile.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/MissileProjectile.prefab.meta	(date 1754430385629)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 5615e4673e17c104384d4c8493c5d42b
+guid: 1a4f21dcee4df52499447e97950d04b3
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/AsteroidController.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: a4a07ada72adcc44ba51f616ad29384b\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/AsteroidController.prefab.meta b/Assets/Prefabs/AsteroidController.prefab.meta
--- a/Assets/Prefabs/AsteroidController.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/AsteroidController.prefab.meta	(date 1754430385617)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: a4a07ada72adcc44ba51f616ad29384b
+guid: 091d7f72950dcf340b8bee6caee4a35d
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/LaserProjectile.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: d4f6577bb30c48945a17243a523023ff\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/LaserProjectile.prefab.meta b/Assets/Prefabs/LaserProjectile.prefab.meta
--- a/Assets/Prefabs/LaserProjectile.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/LaserProjectile.prefab.meta	(date 1754430385622)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: d4f6577bb30c48945a17243a523023ff
+guid: 5d04acbed1c761940898beece0e9d6cd
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/Weapons.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 6b9f7584524aea1439de28cef97b66d0\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/Weapons.prefab.meta b/Assets/Prefabs/Weapons.prefab.meta
--- a/Assets/Prefabs/Weapons.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/Weapons.prefab.meta	(date 1754430385648)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 6b9f7584524aea1439de28cef97b66d0
+guid: 0850e24d00f0a904f9c8028661cd1bbd
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/SparksVFX.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: ffd372f043560f0489da9b0c84e4acac\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/SparksVFX.prefab.meta b/Assets/Prefabs/SparksVFX.prefab.meta
--- a/Assets/Prefabs/SparksVFX.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/SparksVFX.prefab.meta	(date 1754430385639)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: ffd372f043560f0489da9b0c84e4acac
+guid: a99288d254c16ca41a610efd29d7ce66
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/Asteroid3D.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: f0cce3a0c284df24db1283dd96c8c4b0\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/Asteroid3D.prefab.meta b/Assets/Prefabs/Asteroid3D.prefab.meta
--- a/Assets/Prefabs/Asteroid3D.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/Asteroid3D.prefab.meta	(date 1754430385614)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: f0cce3a0c284df24db1283dd96c8c4b0
+guid: 37a8aafb4cb457d41b92cd0da0255ecf
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/UILockOnIndicator.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 8b42cad1f62b7f64b96a4ce6d2329c60\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/UILockOnIndicator.prefab.meta b/Assets/Prefabs/UILockOnIndicator.prefab.meta
--- a/Assets/Prefabs/UILockOnIndicator.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/UILockOnIndicator.prefab.meta	(date 1754430385644)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 8b42cad1f62b7f64b96a4ce6d2329c60
+guid: 99bd765103e23244e854da3e1f795a3e
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/Prefabs/MissileAmmoIcon.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\nguid: 0c6ae3f1d3680b34aa10e39998429f1c\nPrefabImporter:\n  externalObjects: {}\n  userData: \n  assetBundleName: \n  assetBundleVariant: \n
===================================================================
diff --git a/Assets/Prefabs/MissileAmmoIcon.prefab.meta b/Assets/Prefabs/MissileAmmoIcon.prefab.meta
--- a/Assets/Prefabs/MissileAmmoIcon.prefab.meta	(revision c4b9374a2c1d998794b4003d5bfcd52f2c4f1d14)
+++ b/Assets/Prefabs/MissileAmmoIcon.prefab.meta	(date 1754430385627)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 0c6ae3f1d3680b34aa10e39998429f1c
+guid: a4ff64d3c369ef348a4569ad46ea6ff5
 PrefabImporter:
   externalObjects: {}
   userData: 
